### PR TITLE
Clarify usage on rdiff(1) man page.

### PR DESCRIPTION
**Pull request by AaronM04**

### CODE REVIEW

**Comment by dbaarda** [comment]
Is that clear enough for you, or do you want me to add a description of signature and delta files?

**Comment by dbaarda** [comment]
Sorry, I didn't realize my comment was stuck in a pending review until now, so I submitted it. Sorry for the confusion.

**Comment by AaronM04** [comment]
@dbaarda your update looks much clearer. Thanks!

Let me know if there's anything else I need to do.

File: doc/rdiff.1 [hunk 1]
```diff
@@ -26,13 +26,17 @@ rdiff \- compute and apply signature-based file differences
...
 .fi
 .SH USAGE
 You can use \fBrdiff\fP to update files, much like \fBrsync\fP does.
 However, unlike \fBrsync\fP, \fBrdiff\fP puts you in control.  There
 are three steps to updating a file: \fBsignature\fP, \fBdelta\fP, and
-\fBpatch\fP.
+\fBpatch\fP. To generate a \fIdelta-file\fP which can be used in combination
+with \fIold-file\fP to re-generate \fInew-file\fP, use the \fBsignature\fP then
+\fBdelta\fP subcommands. The \fIsignature-file\fP file can be discarded
+afterward. To re-generate \fInew-file\fP, use the \fBpatch\fP
```
[the remark this answers is not in the document]
Comment (dbaarda) [reply doc/rdiff.1]: As per AaronM04's reply my modified version of this seems OK, so I'm going to merge this now.

**Pull request merged**